examples/cluster/plot_dict_face_patches.py

```diff
@@ -44,7 +45,10 @@
 buffer = []
 t0 = time.time()
 
+# %%
 # The online learning part: cycle over the whole dataset 6 times
+# --------------------------------------------------------------
```

Comment:
This cell then would be renamed "Learn the dictionary of images" and the line ``kmeans = MiniBatchKMeans(n_clusters=81, random_state=rng, verbose=True)`` should be move here